packages/std/src/query/query_response.rs

```diff
@@ -1,4 +1,4 @@
-use std::fmt::Debug;
+use core::fmt::Debug;
```

Comment:
Would that be a good candidate for the prelude as well? Display and Debug

Reply:
The problem with that is that their functions collide with each other because both have a `fn fmt(...)`, so I would prefer not to. It would be possible to export both in a `fmt` sub-module, but at that point it's just easier to import `core::fmt::Debug` instead of `crate::prelude::fmt::Debug`